cmd/kube-scheduler/app/server.go

```diff
@@ -74,13 +74,16 @@ func NewSchedulerCommand(registryOptions ...Option) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use: "kube-scheduler",
-		Long: `The Kubernetes scheduler is a policy-rich, topology-aware,
-workload-specific function that significantly impacts availability, performance,
-and capacity. The scheduler needs to take into account individual and collective
-resource requirements, quality of service requirements, hardware/software/policy
-constraints, affinity and anti-affinity specifications, data locality, inter-workload
-interference, deadlines, and so on. Workload-specific requirements will be exposed
-through the API as necessary. See [scheduling](https://kubernetes.io/docs/concepts/scheduling/)
+		Long: `The Kubernetes scheduler watches for newly created Pods that
+have no Node assigned. For every Pod that the scheduler discovers, the
+scheduler becomes responsible for finding the best Node for that Pod to run on.
+The scheduler is a policy-rich, topology-aware, workload-specific function that
+significantly impacts availability, performance, and capacity. The scheduler
+needs to take into account individual and collective resource requirements,
+quality of service requirements, hardware/software/policy constraints, affinity
+and anti-affinity specifications, data locality, inter-workload interference,
+deadlines, and so on. Workload-specific requirements will be exposed through
```

Comment:
Things I'd want readers to learn, instead of this:
- you can have more than one scheduler in your cluster (for high availability)
- you can have more than one scheduler in your cluster (if you deploy multiple, different schedulers, and configure things)
- kube-scheduler is part of the control plane
- kube-scheduler works by first filtering out nodes where the Pod can't possibly be placed, and then scoring what's left to find a good (good enough?) fit
   - https://kubernetes.io/docs/concepts/configuration/scheduling-framework/ talks about the full cycle, but I'd not mention that
- kube-scheduler considers multiple constraints when scheduling, but I wouldn't list which ones (at least not here)

Reply:
I've updated this with a more substantial rewrite to incorporate these points while still being short enough to understand.